tck/src/main/java/org/eclipse/microprofile/concurrency/tck/ThreadContextTest.java

```diff
@@ -116,6 +116,75 @@ public void builderForThreadContextIsProvided() {
         Assert.assertNotNull(ThreadContext.builder(),
                 "MicroProfile Concurrency implementation does not provide a ThreadContext builder.");
     }
+    
+    /**
+     * Verify that the MicroProfile Concurrency ThreadContext implementation clears context
+     * types that are not configured under propagated, unchanged, or cleared.
+     * @throws TimeoutException 
+     * @throws ExecutionException 
+     * @throws InterruptedException 
+     */
+    @Test
+    public void clearUnspecifiedContexts() throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
This seems identical to the prior test (excepting that builder is used instead of CDI/MPConfig) so the same comments apply.